### DISCUSSION TITLE
Adding ssh access to VM  with already configured db service.

### DISCUSSION THREAD

**Discussion by SergeyRim** [gravitational/teleport#36950] [Q&A]
Hello.
I'm using a teleport cloud. And have already configured database service on my vm host (I have configured it using guided enroll new resources from teleport web ui). Everything is working fine.
Now I'm trying to add an ssh access to the same vm using https://goteleport.com/docs/server-access/getting-started guide.
I have create new token with 'db' and 'node' types:
> tctl tokens add --type=node,db

and adjusted /etc/teleport.yaml with new token and ssh_service: enabled: "yes".

After restarting teleport service. database service continue to working fine, but my ssh service is not appears in my resources. There is also a log message in teleport logs:

> Inventory control stream failed: inventory control stream closed: control stream not configured to support ssh server heartbeats inventory/inventory.go:196

Googled for it with no luck.
Can someone please help me what I'm doing wrong?

Thanks.

**Comment by stevenGravy** [comment]
Please indicate the version you are working and share the configuration within your Teleport config.

**Comment by zmb3** [comment]
The token in teleport.yaml is only used on the first join attempt. If you have existing host certs it will be ignored. 

Did you remove /var/lib/teleport prior to restarting the service with the new token?
